webview_panels/src/modules/documentationEditor/components/docGenerator/BulkGenerateButton.tsx

```
@@ -100,63 +118,92 @@ const BulkGenerateButton = (): JSX.Element => {
+                          <Button
+                            color="secondary"
+                            onClick={() => {
+                              close();
+                              void onOptionSelect(option.value);
+                            }}
+                          >
+                            {option.label}
+                          </Button>
+                        </li>
+                      ))}
+                    </>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)